espaloma/data/tests/test_qcarchive.py

```diff
@@ -12,3 +12,67 @@ def test_get_graph():
     collection, record_names = qcarchive_utils.get_collection(client)
     record_name = record_names[0]
     graph = qcarchive_utils.get_graph(collection, record_name)
+
+
+def test_get_torsiondrive():
```

Comment:
Great that we are starting to test these things! A quick docstring here about what the test is doing would be desirable. Thanks!